R/utils_render_grid.R

```diff
@@ -948,14 +948,14 @@ render_grid_svg <- function(label, style, margin) {
   width <- height <- NULL
 
   # Try if any height is declared in style attribute
-  if (any(grepl("^height:", svg_style))) {
+  if (any(startsWith(svg_style, "height:"))) {
     height <- gsub("^height:", "", svg_style[grep("^height:", svg_style)]) %>%
       parse_fontsize(style$text_gp$fontsize) %>%
       grid::unit(.grid_unit)
   }
 
   # Try if any width is declared in style attribute
-  if (any(grepl("^width:", svg_style))) {
+  if (any(startsWith(svg_style, "width:"))) {
```

Comment:
```suggestion
  if (any(grepl("^width:", svg_style))) {
```

Reply:
I'd expect an error from `startsWith()` returning `NA` (--> `if (NA)` --> error), is there a `tryCatch()` somewhere obscuring that?

What about `any(na.rm=TRUE)` instead?

(not worth more than a few seconds' effort, so let's just revert if it requires deeper thought/investigation...)

Reply:
You are right. It is better to investigate locally, by running the tests in test-as_gtable.R and tracking down the issue manually to see where a NA or length 0  is introduced.

Reply:
Let's leave it as a future exercise :)